scripts/content-utils/content-utils.ts

```diff
@@ -0,0 +1,295 @@
+): CardData {
+    return {
+        ...template,
+        ...override,
+    }
+}
+
+/**
+ * Get the unsplash image URL for a given id
+ *
+ * @param id The unsplash image id
+ */
+export function unsplashImage(id: string): string {
+    return `https://images.unsplash.com/photo-${id}?fit=crop&w=800&q=60`
+}
+
+/**
+ * Get the Pexels image URL for a given id
+ *
+ * @param id The pexels image id
+ */
+export function pexelsImage(id: string): string {
+    return `https://images.pexels.com/photos/${id}/pexels-photo-${id}.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=300`
+}
+
+// Credit: https://gist.github.com/mathewbyrne/1280286
+function slugify(text: string): string {
+    return text
+        .toString()
+        .toLowerCase()
+        .replace(/\s+/g, "-") // Replace spaces with -
+        .replace(/[^\w\-]+/g, "") // Remove all non-word chars
+        .replace(/\-\-+/g, "-") // Replace multiple - with single -
+        .replace(/^-+/, "") // Trim - from start of text
+        .replace(/-+$/, "") // Trim - from end of text
+}
+
+/**
+ * Create a WorldQuery.
+ *
+ * WorldQueries are used to define when cards should be available, or when events should trigger.
+ *
+ * @param state A set of WorldStateRanges that need to exist when this query should match
+ * @param flags A set of flags that need to exist when this query should match
+ */
+export function worldQuery(
+    state: WorldQuery["state"] = {},
+    flags: WorldQuery["flags"] = {},
+): WorldQuery {
+    return {
+        state,
+        flags,
+    }
+}
+
+export type Modifier = CardActionData["modifiers"][number]
+
+/**
+ * Create a swipeAction by combining a description with one or more modifiers.
+ *
+ * @param description A short text to explain one of the alternatives in a swipe decision.
+ * @param modifiers One or more modifiers that should be applied when the player choose this course of action.
+ */
+export function action(
+    modifiers: Modifier | Modifier[],
+    description?: string,
+): CardActionData {
+    return {
+        description,
+        modifiers: Array.isArray(modifiers) ? modifiers : [modifiers],
+    }
+}
+
+/**
+ * Easily create a `set` modifier, to be used within an action
+ *
+ * Modifiers update the game state and flags
+ *
+ * @param state The state to modify
+ * @param flags The flags to modify
+ */
+export const setModifier = modifier("set")
+
+/**
+ * Easily create an `add` modifier, to be used within an action
+ *
+ * Modifiers update the game state and flags
+ *
+ * @param state The state to modify
+ * @param flags The flags to modify
+ */
+export const addModifier = modifier("add")
+
+/**
+ * Easily create a `replace` modifier, to be used within an action
+ *
+ * Modifiers update the game state and flags
+ *
+ * @param state The state to modify
+ * @param flags The flags to modify
+ */
+export const replaceModifier = modifier("replace")
+
+/**
+ * Create an modifier factory
+ *
+ * @param type The modifier type to use
+ */
+export function modifier(
```

Comment:
This new factory adds one modifier at a time, moving the logic to combining multiple modifiers for a single action to the `action()` creator itself.

Really nice to keep the curried functions to get modifier types, while also adding support for multiple modifiers. Took some thought but this felt like a good solution when I used it!